src/Scripts/Character_Scripts/Character.gd

```diff
@@ -118,11 +120,14 @@ func change_follow(formation: String):
 	self.party_data["sequence_formation"] = formation
 	
 func change_speed(speed_string: String):
-	var default_speed: float = 60
 	var new_speed = float(speed_string)
 	$AnimatedSprite.set_speed_scale(new_speed/default_speed) 
 	self.speed = float(new_speed)
 
+func restore_default_speed():
+	$AnimatedSprite.set_speed_scale(default_speed/default_speed) 
```

Comment:
1. Isn't this just setting the speed scale to 1? 
2. How does this work? Does it just multiply each animation fps by the scalar?
3. This works in the general case, but if you have time may want to make a way to set the fps of specific animations and then restore their defaults, since some animations won't necessarily be related to the walking speed of the character.